.github/ISSUE_TEMPLATE/als-----wil-ik----.md

```diff
@@ -20,14 +20,15 @@ about: Beschrijving van een feature/user story
 - [ ] Is klein genoeg (maximaal 1/5 van sprint)
 - [ ] Product Owner akkoord en voorzien van prioriteit (mag alleen afgevinkt worden door PO)
 - [ ] Idee hebben van hoe deze user story kan worden gedemonstreerd.
-- [ ] _**Userstory is voorzien van een analyse (met daarin architectuur, IM, technische beslissingen afgestemd met team)**_
+- [ ] Userstory is voorzien van een analyse (met daarin architectuur, IM, technische beslissingen afgestemd met team)
```

Comment:
Waarom blijft dit er in? Bij het merendeel van de User Stories is het er niet. Ik voeg bij Archiveren, Notificaties, etc. wel een soort architectuur/visie/oplossingsrichting toe maar het is niet zo dat Squad Architectuur of onze architect dit voor alle US gaat doen toch? Zelfde voor IM?

Afstemmen met het team, hmmm, doen we dat?

Ik pleit om dit scherper te maken. Of we moeten dit ook echt gaan doen.

Reply:
Ja, kan er uit.

Reply:
scherp